### PR TITLE
add packit

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
blocked on https://github.com/theforeman/foreman-packaging/pull/11336

File: .packit.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - ruby -rrubygems -e 'puts Gem::Specification::load(Dir.glob("*.gemspec").first).version'
+  create-archive:
+    - gem build smart_proxy_pulp.gemspec
+    - bash -c "ls -1t ./smart_proxy_pulp-*.gem | head -n 1"
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    targets:
+      rhel-8:
```
Comment (ekohl) [inline .packit.yaml:31]: We're about to drop el8. This reminds me we need update all packit configs as well

File: .packit.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rubygem-smart_proxy_pulp.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - rubygem-smart_proxy_pulp.spec
```
Comment (ekohl) [inline .packit.yaml:8]: Mimd fixing the indenting here?

Comment (evgeni) [reply .packit.yaml:8]: If you update the template ;)

Comment (evgeni) [reply .packit.yaml]: https://github.com/theforeman/smart_proxy_plugin_template/pull/2

File: .packit.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    targets:
+      rhel-9:
+        additional_modules: "foreman-devel:el9"
+        additional_repos:
+          - https://yum.theforeman.org/releases/nightly/el9/x86_64/
+          - https://yum.theforeman.org/plugins/nightly/el9/x86_64/
+    module_hotfixes: true
```
Comment (ekohl) [inline .packit.yaml:36]: Do we need this without el8?

Comment (evgeni) [reply .packit.yaml:36]: Need? No.
Will it be useful in the future if we ever decide to flip Ruby to anything other than 3.0 and not have to touch every single packit yaml? Yes :)

**Review by ekohl — APPROVED** [review]

**Pull request merged**